proto/one/operations/model/external/delphi_model_details.proto

```diff
@@ -22,7 +22,7 @@ message DelphiModelDetail  {
     // Comment
     google.protobuf.StringValue comment = 6;
     //list of supported Delphi model modes
-    repeated EnumDelphiModelMode modelModesEnabled = 7;
+    repeated EnumDelphiModelMode modelModes = 7;
```

Comment:
I think this one should stay modelModesEnabled since it maps directly to the modelModes enabled field in the delphiModelExecution class

Reply:
correct.

Reply:
At this point we have the front end aligned with the current shape.  Let's chat if this is a big deal to adapt on the other side of team.  Thanks!